app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainFragment.kt

```diff
@@ -44,8 +30,10 @@ class MainFragment : Fragment() {
                 )
             )
         })
-        viewModel.fetchStocks()
```

Comment:
This function call appears to do nothing here so I removed it

Reply:
Or is it?  Since stocks is an observable, this could be kicking off the read of those data.  I'm reluctant to delete something from someone else's project, unless I'm certain it is not needed.